velox/exec/Operator.h

```diff
@@ -508,7 +508,8 @@ class Operator : public BaseRuntimeStatWriter {
 
     uint64_t reclaim(memory::MemoryPool* pool, uint64_t targetBytes) override;
 
-    void abort(memory::MemoryPool* pool) override;
+    void abort(memory::MemoryPool* pool, std::exception_ptr /* error */)
```

Comment:
```
/*unused*/? not sure
```

Reply:
I think it should be fine. Preserving the name of this parameter makes more sense to me